numeric.c

```diff
@@ -991,7 +1004,6 @@ flo_to_s(VALUE flt)
     }
     else if (isnan(value))
 	return rb_usascii_str_new2("NaN");
-
```

Comment:
In general, please avoid making non-documentation changes during documentation updates.

Reply:
> 
> 
> Looks mostly good. I think for the basic mathematical operators, we should reduce the number of examples, see inline comments.

Don't agree on this one.  I think it's worth reminding that the result may not be a Float, and in particular that it may be Infinity, -Infinity, or NaN.

Reply:
> 
> 
> In general, please avoid making non-documentation changes during documentation updates.

A mistake; will correct.  Thanks for noticing.

Reply:
I didn't object to the example showing the return of Complex result.  I don't think it makes sense to include Infinite/NaN example results for every method.  Those should be considered similar to exceptions, and we don't provide examples for exceptions per the documentation guide.